src/renderer/components/Input/InputDelegate.vue

```diff
@@ -0,0 +1,362 @@
+<template>
+  <MenuDropdown
+    ref="dropdown"
+    :items="suggestions"
+    :value="dropdownValue"
+    :pin-to-input-width="true"
+    class="InputDelegate__MenuDropdown"
+    @select="onDropdownSelect"
+    @click="focus"
+  >
+    <InputField
+      slot="handler"
+      :label="label"
+      :helper-text="error || getHelperText()"
+      :is-dirty="$v.model.$dirty"
+      :is-focused="isFocused"
+      :is-invalid="invalid"
+      class="InputDelegate text-left"
+    >
+      <div
+        slot-scope="{ inputClass }"
+        :class="inputClass"
+        class="flex flex-row"
+      >
+        <input
+          ref="input"
+          v-model="model"
+          :name="name"
+          type="text"
+          class="InputDelegate__input flex flex-grow bg-transparent text-theme-page-text"
+          @blur="onBlur"
+          @focus="onFocus"
+          @click.self.stop
+          @keyup.up="onKeyUp"
+          @keyup.down="onKeyDown"
+          @keyup.esc="onEsc"
+          @keyup.enter="onEnter"
+        >
+        <ButtonModal
+          ref="button-qr"
+          :label="''"
+          class="InputDelegate__qr-button flex flex-no-shrink text-grey-dark hover:text-blue"
+          icon="qr"
+          view-box="0 0 20 20"
+          @click.stop
+        >
+          <template slot-scope="{ toggle, isOpen }">
+            <ModalQrCodeScanner
+              v-if="isOpen"
+              :toggle="toggle"
+              @close="toggle"
+              @decoded="onDecodeQR"
+            />
+          </template>
+        </ButtonModal>
+      </div>
+    </InputField>
+  </MenuDropdown>
+</template>
+
+<script>
+import { required } from 'vuelidate/lib/validators'
+import { ButtonModal } from '@/components/Button'
+import ModalQrCodeScanner from '@/components/Modal/ModalQrCodeScanner'
+import { MenuDropdown } from '@/components/Menu'
+import Cycled from 'cycled'
+import InputField from './InputField'
+import truncate from '@/filters/truncate'
+import _ from 'lodash'
```

Comment:
It might be worth loading specifically which lodash modules we want here instead of all of them